components/navbar/navbar.snip.html

```diff
@@ -57,7 +57,7 @@
     </nav>
     {{/sidebaronly}}
     {{#fixeditems}}
-    <div class="ampstart-headerbar-fixed center m0 p0 flex justify-center nowrap">
+    <div class="ampstart-headerbar-fixed center m0 p0 flex justify-center nowrap {{fixeditems-layout-classes}}">
```

Comment:
Added `fixeditems-layout-classes` here, I think this could be refactored template wide at a later stage so the `layout-classes` can sit inside a `fixeditems` object. However this would be a breaking change, so a separate PR would be more suitable.